src/main/kotlin/com/hibob/academy/unitests/keren/ListManager.kt

```diff
@@ -0,0 +1,78 @@
+package com.hibob.academy.unitests.keren
+
+//**
+//* Exercise Instructions:
+//*
+//* Write tests for addPerson method:
+//*
+//* Test adding a unique person.
+//* Test adding a duplicate person and ensure it throws the expected exception.
+//* Test adding multiple people, checking that the list grows appropriately.
+//*
+//*
+//* Write tests for removePerson method:
+//*
+//* Test removing a person that exists in the list.
+//* Test trying to remove a person that does not exist, ensuring it returns false.
+//* Test the state of the list after multiple add and remove operations.
+//*
+//*
+//* Write tests for getPeopleSortedByAgeAndName method:
+//*
+//* Test with an empty list.
+//* Test with one person.
+//* Test with multiple people to ensure they are sorted first by age, then by name.
+//* Test with edge cases like people with the same name but different ages and vice versa.
+//*
+//*/
+data class Person(val name: String, val age: Int)
+
+data class PeopleStatistics(
+    val averageAge: Double,
+    val youngest: Person,
+    val oldest: Person,
+    val ageCount: Map<Int, Int>
+)
+
+class ListManager {
+    private val people: MutableList<Person> = mutableListOf()
+
+    fun addPerson(person: Person): Boolean {
+        if (people.any { it.name == person.name && it.age == person.age }) {
+            throw IllegalArgumentException("Duplicate person cannot be added.")
+        }
+        return people.add(person)
+    }
+
+    fun getManagersAmount(): Int{
```

Comment:
They are not managers, they are simple people :)
 ListManager means it manages a list of people

Reply:
You're right!